### PR TITLE
Resolve SER-3827

### PR DESCRIPTION
**Pull request by akahan**

### CODE REVIEW

File: src/engine/controllers/projectController.ts [hunk 1]
```diff
@@ -97,18 +97,17 @@ export class ProjectController {
     const name = path.basename(context.config.rootExecutionDir);
     context.logger.debug('start initialize project "' + name + '"');
 
-    const projectData = ProjectController.getProjectData(context);
-    const { extensions, gqlSchema } = projectData;
+    const { extensions, gqlSchema, plugins } = ProjectController.loadProjectData(context);
 
-    context.logger.debug('initialize plugins structure');
-    const pluginPaths = this.loadConfigFile(context).plugins;
+    if (plugins) {
```
Comment (eugeneantonevich) [inline src/engine/controllers/projectController.ts:102]: it's not the same behavior.

File: src/common/context.ts [hunk 2]
```diff
@@ -54,15 +53,19 @@ export class Context {
...
-      format: winston.format.printf((info: TransformableInfo) => {
-        if (info.level === 'info') {
-          return info.message;
-        }
-        if (info.level === 'debug') {
-          return `${chalk.hex(Colors.blue)(info.level)} [${Date.now()}]: ${info.message}`;
-        }
-        return `${chalk.hex(Colors.red)(info.level)}: ${info.message}`;
-      }),
+      format: winston.format.combine(
```
Comment (eugeneantonevich) [inline src/common/context.ts:56]: why do we need this changes?

Comment (akahan) [reply src/common/context.ts:56]: To make it easier to write logs.

File: src/engine/controllers/commandController.ts [hunk 3]
```diff
@@ -69,12 +68,13 @@ export class CommandController {
...
 
-        const time = Date.now() - start;
+        const elapsed = Date.now() - started;
 
         context.logger.info(
-          `${chalk.hex(Colors.green)(command)} done. Time: ${chalk.hex(Colors.green)(
-            time.toLocaleString('en-US'),
-          )} sec.`,
+          '%s done. Started: %s. Elapsed: %s sec.',
+          chalk.green(command),
```
[the remark this answers is not in the document]
Comment (akahan) [reply src/engine/controllers/commandController.ts:75]: To make it easier to write logs.

File: src/engine/controllers/projectController.ts (hunk 1 above)
Comment (akahan) [reply src/engine/controllers/projectController.ts:102]: What exactly do you mean?

**Review by s1owjke — APPROVED** [review]

**Pull request closed without merging**